test/system/030-run.bats

```diff
@@ -736,4 +736,11 @@ EOF
     is "$output" "$random_1" "output matches STDIN"
 }
 
+# rhbz#1902979 : podman run fails to update /etc/hosts when --uidmap is provided
+@test "podman run update /etc/hosts" {
```

Comment:
Please make this test name more descriptive, such as by including something about uidmap.

Reply:
Fixed